src/ci/common.sh

```diff
@@ -15,17 +15,20 @@
 # limitations under the License.
 #
 
+# ignore unused var in this script as this script defines common bits for our other scripts
+# shellcheck disable=SC2034
 
 # deploy snapshot from ONLY this branch
 SNAPSHOT_BRANCH="master"
+PROJECT_NAME="okta-sdk-java"
 
 # Get the slug from the TRAVIS var, or parse the 'origin' remote
 REPO_SLUG=${REPO_SLUG:-${TRAVIS_REPO_SLUG:-$(git remote get-url origin | sed 's_.*\:__; s_.*github.com/__; s_\.git__')}}
 PULL_REQUEST=${PULL_REQUEST:-${TRAVIS_PULL_REQUEST:-true}} # default to true
 BRANCH=${TRAVIS_BRANCH:-"$(git rev-parse --abbrev-ref HEAD)"}
 
 # run the ITs if we have an ENV_VARS are set
-if [ $TRAVIS_SECURE_ENV_VARS = true ] ; then
+if [ "$TRAVIS_SECURE_ENV_VARS" = true ] ; then
```

Comment:
nit: Use variable expansion here as you've done in other places

`"$TRAVIS_SECURE_ENV_VARS"` -> `"${TRAVIS_SECURE_ENV_VARS}"`